.github/workflows/nightly.yml

```diff
@@ -31,31 +31,241 @@ jobs:
           mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=full -DjavaVersion=8 -DoutputDirectory=app -Dgoals="clean package"
           rm -rf app
 
+      - name: Run Archetype for EE 8, SE 8, with Docker
+        run: |
+          mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=full -DjavaVersion=8 -Ddocker=yes -DoutputDirectory=app -Dgoals="clean package"
+          if [ -f app/jakartaee-hello-world/Dockerfile ]; then
+            echo "Dockerfile found, test failed."
+            exit 1
+          fi
+          rm -rf app
+
       - name: Run Archetype for EE 8 Web Profile, SE 8
         run: |
           mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=web -DjavaVersion=8 -DoutputDirectory=app -Dgoals="clean package"
           rm -rf app
 
+      - name: Run Archetype for EE 8 Web Profile, SE 8, with Docker
+        run: |
+          mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=web -DjavaVersion=8 -Ddocker=yes -DoutputDirectory=app -Dgoals="clean package"
+          if [ -f app/jakartaee-hello-world/Dockerfile ]; then
+            echo "Dockerfile found, test failed."
+            exit 1
+          fi
+          rm -rf app
+
+      - name: Run Archetype for EE 8 Core Profile, SE 8
```

Comment:
Since we know the Core Profile selection will fail based on the Jakarta EE version and not the other parameters, I think we can reduce the number of tests a little. I think lets do this particular check only once for each Jakarta EE version - so probably just keep this one and not the rest of it's kind for a Jakarta EE version.